### DISCUSSION TITLE
Using VPNs to bypass carrier restrictions

### DISCUSSION THREAD

**Discussion by bobradamus** [Ysurac/openmptcprouter#3683] [Q&A]
Hey, I’ve spent the past six hours trying to get this to work with no success.

I’ve been using OMR for livestreaming for several months now, and it’s been working well, however recently two of my 5G cellular connections are displaying the ‘MPTCP is blocked by network’ on the status page.

I’m using three Teltonika RUTX50 routers, each with a different network provider SIM card in each and all three achieving 5G connections successfully.

I tried enabling the option to use a VPN over the WAN, but that breaks the wan connection with more configuration needed and, while I’m not an idiot with technology, without some instruction I can’t get it to work - at all.

It seems to create OpenVPN connections for each one you enable, but these aren’t setup completely and then need further configuration. I’ve tried to do this myself scouring the internet and other questions / issues from people on here however I cannot get it to work.

Has anyone else done this who’d be willing to just list the steps they went through to get it to work. I think otherwise I’m gonna have to write this off and use a different way of bonding the connections and delivering the livestream over HLS instead of RTMP using OMR.

Please help!!

**Comment by Ysurac** [comment]
Before enabling MPTCP over VPN, did you check if aggregation is working ?

**Reply by bobradamus** [reply]
ah, so it is actually working, I’ll ignore that error message in future! 😂
